src/modules/esl-footnotes/core/esl-footnotes-data.ts

```diff
@@ -36,11 +36,15 @@ export function compileFootnotesGroupedList(notes: ESLNote[]): FootnotesItem[] {
   return groupedList;
 }
 
-/** Sort notes list */
+/** Sorts notes list */
 export function sortFootnotes(notes: ESLNote[]): ESLNote[] {
-  return notes.sort((note1: ESLNote, note2: ESLNote) => {
-    const rect1 = note1.getBoundingClientRect();
-    const rect2 = note2.getBoundingClientRect();
-    return rect1.top - rect2.top || rect1.left - rect2.left;
+  return notes.sort((note1: ESLNote, note2: ESLNote): number => {
+    if (note1 === note2) return 0;
+    // eslint-disable-next-line no-bitwise
+    if (note1.compareDocumentPosition(note2) & 2) {
```

Comment:
```suggestion
    if (note1.compareDocumentPosition(note2) & Node.DOCUMENT_POSITION_PRECEDING) {
```

Reply:
Or mb better comment... because from one side not sure that it will be minified, but from another, it's not clear what is going on without spec.